src/controllers/queryRunner.ts

```diff
@@ -679,6 +660,74 @@ export default class QueryRunner {
         }
         await p;
 
+        this.constructCopyString(
+            copyString,
+            rowIdToRowMap,
+            rowIdToSelectionMap,
+        );
+
+        await this.writeStringToClipboard(copyString);
+    }
+
+    public async exportCellsToClipboard(
+        data: DbCellValue[][],
+        batchId: number,
+        resultId: number,
+        selection: ISlickRange[],
+        headersFlag,
+    ) {
+        let copyString = "";
+        if (headersFlag) {
+            copyString = this.addHeadersToCopyString(
+                copyString,
+                batchId,
+                resultId,
+                selection,
+            );
+        }
+
+        // create a mapping of rows to selections
+        let rowIdToSelectionMap = new Map<number, ISlickRange[]>();
+        let rowIdToRowMap = new Map<number, DbCellValue[]>();
```

Comment:
A selection potentially spans multiple rows, right?  So will this map have multiple rowId entries pointing to the same ISlickRange?

Reply:
Yes that's correct